roop/processors/frame/face_enhancer.py

```diff
@@ -116,6 +90,53 @@ def restore_face(face_in_tensor) -> Any:
     return convert_tensor_to_image(face_in_tensor)
 
 
+def enhance_face_in_frame(face_list):
+    face_enhanced_list = []
+    for  _, cropped_face in enumerate(face_list):
+        face_in_tensor = normalize_face(cropped_face)
+        face_enhanced = restore_face(face_in_tensor)
+        face_enhanced_list.append(face_enhanced)
+    return face_enhanced_list
+
+
+def paste_face_back(temp_frame, face_enhanced_list, matrix_list):
+    target_img = temp_frame
+    for i, face_enhanced in enumerate(face_enhanced_list):
```

Comment:
This method has to be shrinked down or splitted into more functions that describe the individual step being done to either faces, frames or a matrix